diff --git a/linux/vrouter_mod.c b/linux/vrouter_mod.c
index 6bdd6f7..06ebc43 100644
--- a/linux/vrouter_mod.c
+++ b/linux/vrouter_mod.c
@@ -91,6 +91,26 @@ extern int lh_gro_process(struct vr_packet *, struct vr_interface *, bool);
 static void lh_reset_skb_fields(struct vr_packet *pkt);
 static unsigned int lh_get_cpu(void);
 
+
+
+
+#define _HAVE_ARCH_IPV6_CSUM
+extern __wsum
+__csum_ipv6_magic(const struct in6_addr *saddr, const struct in6_addr *daddr, __be32 len,
+		__be32 proto, __wsum sum);
+
+static inline __sum16
+csum_ipv6_magic(const struct in6_addr *saddr, const struct in6_addr *daddr,
+		__u32 len, __u8 proto, __wsum sum)
+{
+	return csum_fold(__csum_ipv6_magic(saddr, daddr, htonl(len),
+					   htonl(proto), sum));
+}
+
+
+
+
+
 static int
 lh_printk(const char *format, ...)
 {
